api/execution/src/main/java/quarks/execution/mbeans/JobMXBean.java

```diff
@@ -150,4 +152,13 @@ static public Health fromString(String health) {
      * @return a JSON-formatted string representing the running graph. 
      */
     String graphSnapshot();
+    
```

Comment:
I'm thinking of a JobMXBean state as a read-only attribute, reflecting changes initiated by the Etiao runtime.  I'm wondering what is the use case for initiating a job state change from a JMX client?